package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A helpful wrapper for dogstatsd.",
   "main": "index.js",
   "scripts": {
-    "test": "jshint index.js lib/ && lab -v -c test/*"
+    "test": "jshint index.js lib/ && jsinspect lib/ && lab -v -c test/*",
+    "jsinspect": "jsinspect lib/"
```

Comment:
the lint and inspect commands should be their own and invoked using `npm run ___` - that way you can change the script value when you need to and not have duplicated commands...